test/uk/gov/hmrc/apprenticeshiplevy/controllers/sandbox/SandboxTestDataControllerSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apprenticeshiplevy.controllers.sandbox
+
+import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.Configuration
+import uk.gov.hmrc.apprenticeshiplevy.controllers.sandbox.data.SandboxTestDataController
+
+class SandboxTestDataControllerSpec extends PlaySpec with GuiceOneAppPerSuite {
```

Comment:
do you need GuiceOneAppPerSuite?

Reply:
No - I added this in and moved away from loading the App. Ill remove it